lib/aggregation/accumulator/package.json

```diff
@@ -49,6 +49,7 @@
     "abacus-urienv": "file:../../utils/urienv",
     "abacus-webapp": "file:../../utils/webapp",
     "abacus-yieldable": "file:../../utils/yieldable",
+    "abacus-dataflow-reducer-config-factory": "file:../dataflow-reducer-config-factory",
```

Comment:
Can we change the name to 'dataflow-reducer-config'?

Reply:
We are going to break "abacus" prefix convention

Reply:
We can then rename it to abacus-dataflow-config or abacus-reducer-config